frame/staking/src/tests.rs

```diff
@@ -4593,6 +4593,38 @@ fn min_commission_works() {
 	})
 }
 
+#[test]
```

Comment:
If there's already a test along the lines of "validate works", checking the event there would've been enough.

Reply:
Agreed, the event test can go inside min_commission_works. Will re-test this shortly